files/scripts/syncd.sh

```diff
@@ -44,11 +44,16 @@ function startplatform() {
 }
 
 function waitplatform() {
-    
+
+    BOOT_TYPE=`getBootType`
     if [[ x"$sonic_asic_platform" == x"mellanox" ]]; then
-        debug "Starting pmon service..."
-        /bin/systemctl start pmon
-        debug "Started pmon service"
+        if [[ x"$BOOT_TYPE" != x"cold" ]]; then
```

Comment:
Same comment as on lldp PR:
Minor: This is correct, but it may be better to match against whitelist BOOT_TYPEs (fast, warm, fastfast).
This is so that this condition does not incorrectly evaluate to true for any other boottype that gets added in future.